cloudos_cli/__main__.py

```diff
@@ -276,9 +383,21 @@ def run(apikey,
         verbose,
         request_interval,
         disable_ssl_verification,
-        ssl_cert):
+        ssl_cert,
+        profile):
     """Submit a job to CloudOS."""
-    cloudos_url = cloudos_url.rstrip('/')
```

Comment:
@danielboloc I've tested: you cannot remove this (here and in all the places you removed it). If the user have not configured a profile yet, this is still necessary

Reply:
I have added it again to all cloudos url variables, thanks for the catch